app/src/main/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryImpl.kt

```diff
@@ -1144,7 +1144,7 @@ class VaultRepositoryImpl(
 
         // Return if local cipher is more recent
         if (localCipher != null &&
-            localCipher.revisionDate.epochSecond > revisionDate.toEpochSecond()
+            localCipher.revisionDate.epochSecond > (revisionDate?.toEpochSecond() ?: 0)
```

Comment:
Is this a fair assumption to make? I'd assume if the data was `null` because it came from a push notification lacking that data that it was probably _more_ recent than anything you have stored locally.